src/Service/Terminal/Service_Terminal.rei

```diff
@@ -36,3 +36,5 @@ module Sub: {
 module Effect: {let input: (~id: int, string) => Isolinear.Effect.t(msg);};
 
 let handleExtensionMessage: ExtHostClient.Terminal.msg => unit;
+
+let getScreenOpt: int => option(ReveryTerminal.Screen.t);
```

Comment:
Probably from a merge (which makes PRs a bit hard to review sometimes, btw), but as we've discussed recently the `Opt` is unnecessary.

Reply:
Ah, right, I added this prior to our discussion earlier - fixed now 👍